app/loader.cpp

```diff
@@ -63,6 +63,8 @@ bool Loader::load( const QString &filePath )
   }
 
   emit loadingStarted();
+  QFile flagFile( loadingFile() );
+  flagFile.open( QIODevice::WriteOnly );
```

Comment:
don't you need to immediately close it, so it actually stores/flushes it to the disk?